pkg/porter/stamp.go

```diff
@@ -45,7 +46,23 @@ func (p *Porter) IsBundleUpToDate(opts bundleFileOptions) (bool, error) {
 			return false, errors.Wrapf(err, "could not load stamp from %s", opts.CNABFile)
 		}
 
-		converter := configadapter.NewManifestConverter(p.Context, p.Manifest, nil)
+		installedMixins, err := p.ListMixins()
```

Comment:
This logic here would be good to move into a new function on `p`, such as `getUsedMixins`, so that we don't get drift between these two places where we use it (stamp and buildBundle) and would let us write a unit test for it.

Reply:
Makes sense. I will move to separate function